src/graphql/data/fetchGraphQL.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Helpful Resources
+ * https://github.com/sibelius/create-react-app-relay-modern/blob/master/src/relay/fetchQuery.js
+ * https://github.com/relay-tools/relay-compiler-language-typescript/blob/master/example/ts/app.tsx
+ */
+//import { aws4Interceptor } from 'aws4-axios'
+import { Variables } from 'react-relay'
+import { GraphQLResponse, ObservableFromValue, RequestParameters } from 'relay-runtime'
+
+// Define a function that fetches the results of a request (query/mutation/etc)
+// and returns its results as a Promise:
```

Comment:
i find this comment a bit confusing because the return type of the function is not a Promise

Reply:
`export type ObservableFromValue<T> = Subscribable<T> | Promise<T> | T;`

Reply:
I think we could type it as just `Promise<GraphQLResponse>` and not lose anything